doc/release/RELEASE-NOTES.md

```diff
@@ -17,7 +17,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * *Describe scheme changes here.*
 
 ### Code/API changes
-* *Describe code/API changes here.*
+* [OSDEV-1346](https://opensupplyhub.atlassian.net/browse/OSDEV-1346) - Disabled null values from the response of the OpenSearch. Disabled possible null `os_id`, `claim_id` and `source` from PATCH response.
```

Comment:
> from PATCH response.

For future reference, could you please also include the exact name of the endpoint? Over several months, it can become difficult to remember which PATCH request is being referred to.

Reply:
I've updated this info, thank you.